src/DataFactory/DataFactoryV2.Test/SessionRecords/Microsoft.Azure.Commands.DataFactoryV2.Test.IntegrationRuntimeTests/TestSelfHostedIntegrationRuntime.json

```diff
@@ -1,106 +1,99 @@
 {
   "Entries": [
     {
-      "RequestUri": "/subscriptions/c39dce18-cead-4065-8fb1-3af7683a5038/resourcegroups/ps2452?api-version=2016-09-01",
-      "EncodedRequestUri": "L3N1YnNjcmlwdGlvbnMvYzM5ZGNlMTgtY2VhZC00MDY1LThmYjEtM2FmNzY4M2E1MDM4L3Jlc291cmNlZ3JvdXBzL3BzMjQ1Mj9hcGktdmVyc2lvbj0yMDE2LTA5LTAx",
+      "RequestUri": "/subscriptions/b371d9e7-d3c2-4b1a-83ec-84e1f50c2222/resourcegroups/ps5033?api-version=2016-09-01",
```

Comment:
In the test case, api version is 2018-06-01

/subscriptions/b371d9e7-d3c2-4b1a-83ec-84e1f50c2379/resourceGroups/ps5033/providers/Microsoft.DataFactory/factories/ps1068?api-version=2018-06-01